### DISCUSSION TITLE
About Header List

### DISCUSSION THREAD

**Discussion by Gaobang100** [vuepress-theme-hope/vuepress-theme-hope#2569] [Q&A] [answered]
在我的电脑屏幕上，只有当我将浏览器缩放调整到80%以下，右侧才会显示标题列表。
将标题列表放在侧边栏比较杂乱，我应该如何改进？
![image](https://user-images.githubusercontent.com/105471582/208847669-897e200d-d306-4880-94e7-a5e9cd8615fb.png)
![image](https://user-images.githubusercontent.com/105471582/208847705-5677f375-17d7-43d8-ab64-c3ed789b3e4e.png)

**Comment by Mister-Hope** [comment, marked as answer]
建议买一个分辨率高点的显示器，或者降低你的高分辨率显示器的缩放比例。

对于一个横向渲染像素不超过1440px的设备，我们的选择就是使用标准布局，这可以为用户带来较好的效果。

如果你的确有需要，你可以修改样式里的断点值将其降低。

**Reply by Gaobang100** [reply]
感谢回答，这很有效。
为了方便有同样问题的人，我找到了文档中关于此问题的部分，附在下面
https://vuepress-theme-hope.github.io/v2/zh/config/style.html#config-scss
